### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by andres-condezo**
## Feature/navbar
Changes in this PR:
- Add a navigation bar to:
  - Home Page (List of Cars)
  - Add a Rental (Form)
  - Rentals (List of Rentals)
  - Add a Car
  - Delete a Car

### CODE REVIEW

**Review by RaoAkif — APPROVED** [review]
Hi @andres-condezo 

Your navbar feature is complete! Congratulations! 🎉 But kindly wait for @Retky's review before merging :shipit: 

## To Highlight 💯 
- Smart Logo  ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

Congratulations-Memes-FEATURED

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by Retky — APPROVED** [review]
## Hello @andres-condezo 🔥 
Excellent work! 💯
You have done a great job. 🥳
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit:
Congratulations!!!🎊

# Status: Approved ✔️ 
image
)

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---

**Pull request merged**
